Bc prs

jopeek opened the pull request:
Includes the following contributions:
- #431 
- #432 
- #433 
- #435

costonb committed:
Update moveItem logic to merge item with existing if the character already has a stack

costonb committed:
fix plus minus item quantity permissions

Wrap them in if root owner tags

costonb committed:
Add in ability to split coins

Moved distributeCoins to static method in helper so both code paths went through the same code
Made adjustments to the distributeCoins method so it works properly
Added button to split coins
Added owner check for the manage currency button

jopeek committed:
Merge pull request #431 from costonb/item-merge

merge stacks on loot/buy item

jopeek committed:
Merge pull request #432 from costonb/item-quantity-permissions

fix plus minus item quantity permissions

jopeek committed:
Merge branch 'bc-prs' into split-currency

jopeek committed:
Merge pull request #433 from costonb/split-currency

Add ability to split coins

costonb commented:
Cool.  We've been using this module for a while in our games, but I had a fork with several changes.  The past couple times there's been a major upgrade there's been quite a few merge conflicts, so i figured I'd just submit some PRs.

I do have two more I plan to open.  One for simplifying how the tabs are done.  They provide a static method that can be overridden for that rather than the html manipulation that's happening now.  The other is adding a tab to sell items to the merchant.  That one was working well when you were manually doing the `_prepareItems()` logic, but now that you're just using the super class code I'm having to re-work quite a bit for the player's inventory.  The tab PR I'll open here in the next couple hours, I'll use this branch as the base.  The selling PR will probably not be this week since I mostly only have time and energy to work on this on my days off work.

costonb committed:
Use TABS static variable

Simplify module to override the TABS static variable rather than adjusting html directly in activateListensers method

jopeek committed:
Merge pull request #435 from costonb/simplify-tabs

Use TABS static variable

jopeek committed:
Update module.json
